Client/Packet/Packet.h

```diff
@@ -63,544 +62,549 @@ public :
+			PACKET_GC_USE_POWER_POINT_RESULT,   		//   414
+			PACKET_GC_VISIBLE_FAIL,		//   415
+			PACKET_GC_VISIBLE_OK,		//   416
+			PACKET_GC_WAIT_GUILD_LIST,		//   417
+			PACKET_GC_WAR_LIST,		//   418
+			PACKET_GC_WAR_SCHEDULE_LIST,		//   419
+			PACKET_GC_WHISPER,		//   420
+			PACKET_GC_WHISPER_FAILED,		//   421
+			PACKET_GG_COMMAND,		//   422
+			PACKET_GG_GUILD_CHAT,		//   423
+			PACKET_GG_SERVER_CHAT,		//   424
+			PACKET_GL_INCOMING_CONNECTION,		//   425
+			PACKET_GL_INCOMING_CONNECTION_ERROR,		//   426
+			PACKET_GL_INCOMING_CONNECTION_OK,		//   427
+			PACKET_GL_KICK_VERIFY,		//   428
+			PACKET_GM_SERVER_INFO,		//   429
+			PACKET_GS_ADD_GUILD,		//   430
+			PACKET_GS_ADD_GUILD_MEMBER,		//   431
+			PACKET_GS_EXPEL_GUILD_MEMBER,		//   432
+			PACKET_GS_GUILD_ACTION,     		//   433
+			PACKET_GS_GUILDMEMBER_LOGON,		//   434
+			PACKET_GS_MODIFY_GUILD_INTRO,		//   435
+			PACKET_GS_MODIFY_GUILD_MEMBER,		//   436
+			PACKET_GS_QUIT_GUILD,		//   437
+			PACKET_GS_REQUEST_GUILD_INFO,		//   438
+			PACKET_GTO_ACKNOWLEDGEMENT, 		//   439
+			PACKET_LC_CREATE_PC_ERROR,		//   440
+			PACKET_LC_CREATE_PC_OK,		//   441
+			PACKET_LC_DELETE_PC_ERROR,		//   442
+			PACKET_LC_DELETE_PC_OK,		//   443
+			PACKET_LC_LOGIN_ERROR,		//   444
+			PACKET_LC_LOGIN_OK,		//   445
+			PACKET_LC_PC_LIST,		//   446
+			PACKET_LC_PORT_CHECK,		//   447
+			PACKET_LC_QUERY_RESULT_CHARACTER_NAME,		//   448
+			PACKET_LC_QUERY_RESULT_PLAYER_ID,		//   449
+			PACKET_LC_RECONNECT,		//   450
+			PACKET_LC_REGISTER_PLAYER_ERROR,		//   451
+			PACKET_LC_REGISTER_PLAYER_OK,		//   452
+			PACKET_LC_SELECT_PC_ERROR,		//   453
+			PACKET_LC_SERVER_LIST,		//   454
+			PACKET_LC_VERSION_CHECK_ERROR,		//   455
+			PACKET_LC_VERSION_CHECK_OK,		//   456
+			PACKET_LC_WORLD_LIST,		//   457
+			PACKET_LG_INCOMING_CONNECTION,		//   458
+			PACKET_LG_INCOMING_CONNECTION_ERROR,		//   459
+			PACKET_LG_INCOMING_CONNECTION_OK,		//   460
+			PACKET_LG_KICK_CHARACTER,		//   461
+			PACKET_RC_CHARACTER_INFO,		//   462
+			PACKET_RC_CONNECT_VERIFY,		//   463
+			PACKET_RC_POSITION_INFO,		//   464
+			PACKET_RC_REQUEST_VERIFY,		//   465
+			PACKET_RC_REQUESTED_FILE,		//   466
+			PACKET_RC_SAY,		//   467
+			PACKET_RC_STATUS_HP,		//   468
+			PACKET_SG_ADD_GUILD_MEMBER_OK,		//   469
+			PACKET_SG_ADD_GUILD_OK,		//   470
+			PACKET_SG_DELETE_GUILD_OK,		//   471
+			PACKET_SG_EXPEL_GUILD_MEMBER_OK,		//   472
+			PACKET_SG_GUILD_INFO,		//   473
+			PACKET_SG_GUILD_RESPONSE,   		//   474
+			PACKET_SG_GUILDMEMBER_LOGON_OK,		//   475
+			PACKET_SG_MODIFY_GUILD_INTRO_OK,		//   476
+			PACKET_SG_MODIFY_GUILD_MEMBER_OK,		//   477
+			PACKET_SG_MODIFY_GUILD_OK,		//   478
+			PACKET_SG_QUIT_GUILD_OK,		//   479
+			PACKET_UC_REQUEST_LOGIN_MODE,		//   480
+			PACKET_UC_UPDATE,		//   481
+			PACKET_UC_UPDATE_LIST,		//   482
+			PACKET_CG_ENCODE_KEY,		//   483 add by viva 2008-12-27 : Packet Encode Key
+			PACKET_GC_FRIEND_CHATTING,	//add by viva 
+			PACKET_GC_USE_SKILLCARD_OK, // by viva

```

Comment:
`PACKET_GC_USE_SKILLCARD_OK` is moved ?

https://github.com/opendarkeden/client/pull/9/files#diff-ef122be3d27075fe03b25d7b8834e718dc778526f19bbca8fe9fb6d4c2ae9420L479